### PR TITLE
Adding Initial Authentication Code

### PR DESCRIPTION
**Pull request by mtolrom**
Adding Initial Authentication Code

### CODE REVIEW

**Review by mviencek — CHANGES_REQUESTED** [review]
please add nbproject/project.properties and nbproject/project.xml to gitignore.

why did you remove the style.css file from layout.php?

why did you create your own config page and not use the existing one?

for your inserts and queries, your not protecting against injection.  please escape the posts or use parameterized queries.

please change require(config.php) to use configuration.php instead

**Comment by mviencek** [comment]
i forgot to say that removing     <link rel="stylesheet" href="views/styles/styles.css"> broke several things

**Comment by mviencek** [comment]
link rel="stylesheet" href="views/styles/styles.css"

**Comment by mtolrom** [comment]
I added that config file only for testing. will remove that.
I removed the CSS to let UI people fix it. It's shifting menu items an in incorrect display.
Insert queries will not be used. Back-End team should replace that with store procedures.

**Comment by mviencek** [comment]
no worries  about the sql.  for the rest of it though please remove config.php and remap references to configuration.php also add nbproject/project.properties and nbproject/project.xml to gitignore.

File: views/static/config.php [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<?php
```
Comment (mcontado) [inline views/static/config.php]: remove this file. we already have configuration.php

File: views/static/login.php [hunk 2]
```diff
@@ -0,0 +1,36 @@
+<?php
```
Comment (mcontado) [inline views/static/login.php]: remove this file, Mekone is working on this.

File: views/static/logout.php [hunk 3]
```diff
@@ -0,0 +1,6 @@
+<?php
```
Comment (mcontado) [inline views/static/logout.php]: remove this file, Mekone is working on this.

File: views/static/profile.php [hunk 4]
```diff
@@ -0,0 +1,30 @@
+<?php
```
Comment (mcontado) [inline views/static/profile.php]: remove this file, Mekone is working on this.

File: views/static/registration.php [hunk 5]
```diff
@@ -1,3 +1,42 @@
+<?php
```
Comment (mcontado) [inline views/static/registration.php]: please checkout the latest registration page from the Development. try doing: ```git checkout origin/Development views/static/registration.php```

File: views/static/search_events.php [hunk 6]
```diff
@@ -1,10 +1,12 @@
 <?php
-/**
- * Created by PhpStorm.
- * User: Harry
- * Date: 5/8/2018
- * Time: 3:55 PM
- */
+
```
Comment (mcontado) [inline views/static/search_events.php]: remove this file, Mekone is working on this.

File: views/templates/header.php [hunk 7]
```diff
@@ -1,14 +1,25 @@
...
-            <li class="nav-item">
-                <a class="nav-link" href="?controller=static&action=registration">Sign Up</a>
-            </li>
-            <li class="nav-item">
-                <a href="#" class="nav-link" onclick="document.getElementById('id01').style.display='block'" style="width:auto;">Login</a>
-            </li>
-        </ul>
-    </nav>
-</div>
+<nav class="navbar navbar-expand-sm bg-dark navbar-dark">
```
Comment (mcontado) [inline views/templates/header.php]: remove this file, Mekone is working on this.

**Comment by mtolrom** [comment]
Changes have been pushed.

**Comment by mviencek** [comment]
cool thank you.  i noticed that there are hard coded values in this query
  $query = "INSERT INTO Users(email, password, firstName, lastName, phoneNumber, notificationState, UserTypeId) VALUES ('$emailAddress', '$password', '$firstName', '$lastName', '$phoneNumber', 1, 1)";

was that intentional?
also could you also add the link back to the styles page.  they are aware that there is problems with the file.  they are going to fix them.

File: nbproject/project.xml [hunk 8]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://www.netbeans.org/ns/project/1">
+    <type>org.netbeans.modules.php.project</type>
+    <configuration>
+        <data xmlns="http://www.netbeans.org/ns/php-project/1">
+            <name>PhpProject1</name>
+        </data>
+    </configuration>
+</project>
```
Comment (mviencek) [inline nbproject/project.xml:9]: your nbproject stuff is still coming through

File: nbproject/project.properties [hunk 9]
```diff
@@ -0,0 +1,7 @@
+include.path=${php.global.include.path}
+php.version=PHP_70
+source.encoding=UTF-8
+src.dir=.
+tags.asp=false
```
Comment (mviencek) [inline nbproject/project.properties:5]: your nbproject stuff is still coming through

**Comment by mviencek** [comment]
Can you close your pull request and resubmit it under a different name?  It's causing confusion with Development.  People aren't sure which one to merge with.  Thank you

**Pull request closed without merging**